### PR TITLE
Create popup details

### PR DESCRIPTION
**Pull request by pirc25**
### Create popup details.
In this milestone, I created popup model details, and for it I made:

- Approve HTML, CSS, JS linters.
- Created model using JS and Html.
- Used arrays and objects to save project information to show on the portfolio.
- Used DOM to edit HTML and CSS with JS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -127,7 +127,7 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
           <li>CSS</li>
           <li>Ruby</li>
         </ul>
-        <button type="button" class="see-project-card" id="button-2">
+        <button type="button" class="see-project-card" id="button-2" onclick="openDetail(2)">
```
Comment (danmainah) [inline index.html:130]: - [ ] Kindly ensure your design matches the given design, currently, the layout is a bit different on the pop-up window, also one of your popups doesn't show the projected image.


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image


- [ ] Kindly also check your design at different widths, also the page doesn't fit in the entire window, below is a video showing it, I have indicated the design difference in mobile view compared to the given design kindly fix it.[Click here](https://www.loom.com/share/d43f0fff0f92488282fc4ca000e592e1) to view the video.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @pirc25 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR and Description


# Required Changes ♻️

- [ ] Kindly ensure all your project files are loaded dynamically using javascript, I can see you already have the array but still have the old hardcoded projects in your index.html kindly remove them.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (topeogunleye) [inline index.html:130]: - [ ] As the previous reviewer suggested,  please ensure you follow the Figma design template as your text on the left of your modal should not be floating onto the image on the right
- [ ] Kindly ensure your list of technologies is to the right as seen in the Template design.
- [ ] Kindly ensured that the text in your button is in centered in the middle of the button.

| Template Design  | Your Design |
| ------------- | ------------- |
| Screenshot from 2022-05-20 17-22-00  | Screenshot from 2022-05-20 17-17-08  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @pirc25,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (pirc25) [reply index.html:130]: @topeogunleye Thanks for your review, I am going to change the buttons and the text style. 

However, the technologies is on top of the image. **This template design is not the popUp model.**

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @pirc25 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (topeogunleye) [reply index.html:130]: Okay. That's great. :+1: , sorry for using the design for the modal.

**Pull request merged**